pills/13-callpackage-design-pattern.xml

```diff
@@ -124,7 +124,7 @@
       The "<literal>callPackage</literal>" pattern has simplified our repository a lot. We're able to import packages that require some named arguments and call them automatically, given the set of all packages.
     </para>
     <para>
-      We've also introduced some useful builtin functions that allows us to introspect Nix functions and manipulate attributes. These builtin functions are not usually used when packaging software, rather to provide tools for packaging. That's why they are not documented in the <link xlink:href="http://nixos.org/nix/manual/">nix manual</link>.
+      We've also introduced some useful builtin functions that allows us to introspect Nix functions and manipulate attributes. These builtin functions are not usually used when packaging software, rather to provide tools for packaging. They are documented in the <link xlink:href="http://nixos.org/nix/manual/">nix manual</link>.
```

Comment:
nixos.org has a different hierarchy now and the redirects are iffy plus we can also link to the section directly.
That will break with the next major release most likely, but that's gonna be a while anyways, so I think it doesn't matter.

```suggestion
      We've also introduced some useful builtin functions that allows us to introspect Nix functions and manipulate attributes. These builtin functions are not usually used when packaging software, rather to provide tools for packaging. They are documented in the <link xlink:href="https://nixos.org/manual/nix/stable/#ssec-builtins">nix manual</link>.
```